dcap/types/src/tcb.rs

```diff
@@ -71,13 +78,41 @@ type SgxExtensions = Vec<AttributeTypeAndValue>;
 /// The TCB info provided by the PCK(Provisioning Certification Key) leaf
 /// certificate
 #[derive(Debug, PartialEq)]
-pub struct PckTcbInfo {
-    svns: [u32; 16],
+pub struct TcbInfo {
+    svns: [u32; COMPONENT_SVN_COUNT],
     pce_svn: u32,
     fmspc: [u8; FMSPC_SIZE],
 }
 
-impl TryFrom<&Certificate> for PckTcbInfo {
+impl TcbInfo {
+    /// Get the component SVN values
+    pub fn svns(&self) -> &[u32; COMPONENT_SVN_COUNT] {
+        &self.svns
+    }
+
+    /// Get the PCE SVN value
+    pub fn pce_svn(&self) -> &u32 {
+        &self.pce_svn
+    }
+
+    /// Get the FMSPC value
+    pub fn fmspc(&self) -> &[u8; FMSPC_SIZE] {
+        &self.fmspc
+    }
```

Comment:
I don't have much preference if we should use accessors or directly access members. Other parts of this repo use accessor methods so it may be better to keep that way for consistency.